packages/ngworker/lumberjack/src/lib/logging/lumberjack.service.ts

```diff
@@ -26,7 +25,7 @@ const noReportedLogDriverErrorIndex = -1;
  * base classes to set up predefined loggers unless you need a programmatic
  * API.
  */
-@Injectable({ providedIn: LumberjackRootModule })
+@Injectable({ providedIn: 'root' })
```

Comment:
issue: Must not be injectable without calling `provideLumberjack` or importing `LumberjackModule.forRoot`. Add `LumberjackService` to the `providers` array returned by `provideLumberjack` and `LumberjackRootModule.providers` instead.
```suggestion
@Injectable()
```